products/sentinel_one.py

```diff
@@ -461,17 +471,28 @@ def _process_queries(self):
 
             for tag, queries in self._queries.items():
                 for query in queries:
-                    if query.operator in ('contains', 'containscis', 'contains anycase'):
+                    if query.operator in ('contains', 'containscis', 'contains anycase') and not re.findall(r'(?:\" AND)', query.search_value):
```

Comment:
We don't need to add the regex check since the queries field will always have the `raw` operator

```suggestion
                    if query.operator in ('contains', 'containscis', 'contains anycase'):
```